fotoapparat/src/main/java/io/fotoapparat/hardware/orientation/OrientationUtils.java

```diff
@@ -61,4 +63,25 @@ public static int computeImageOrientation(int screenRotationDegrees,
         return (rotation + 360 + 360) % 360;
     }
 
+    /**
+     * Copied from {@link android.hardware.Camera.Parameters#setRotation(int)}
+     * @param screenOrientationDegrees rotation of the display (as provided by an {@link android.view.OrientationEventListener}) in degrees.
+     * @param cameraRotationDegrees rotation of the camera sensor relatively to device natural
+     *                              orientation.
+     * @param cameraIsMirrored      {@code true} if camera is mirrored (typically that is the case
+     *                              for front cameras). {@code false} if it is not mirrored.
+     * @return clockwise rotation of the image relatively to current device orientation.
+     */
+    public static int computeImageOrientation(int screenOrientationDegrees,
```

Comment:
Tests for this function are missing. Having them would be really nice since it would make clear what kind of result we expect from different inputs. There are already tests for other functions in `OrientationUtils`, you can use them as an example.